layout/save.go

```diff
@@ -27,6 +28,52 @@ func (i *Image) SaveAs(name string, additionalNames ...string) error {
 		ops = append(ops, WithoutLayers())
 	}
 
+	i.Image, err = imgutil.MutateManifest(i, func(mfest *v1.Manifest) {
```

Comment:
This is why I have some doubts about holding attributes, if we just mutate the config file during the set operations do we need to this?